spectrocloud/data_source_pack_simple.go

```diff
@@ -0,0 +1,136 @@
+package spectrocloud
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/spectrocloud/terraform-provider-spectrocloud/pkg/client"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourcePackSimple() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourcePackReadSimple,
```

Comment:
```suggestion
		ReadContext: dataSourcePackReadSimple,
		Description: "",
```
@nikchern this is the global description that needs to be populated for the template so that the reader can get a sense of the resource does. I don't have a good grasp on what a simple pack is, so I'll leave the description to you.